### PR TITLE
init Core Data Stack; add Safe model; add unit tests for Core Data Stack

### PR DESCRIPTION
**Pull request by sche**
sorry, commit message without a prefix;

### CODE REVIEW

File: MultisigTests/Persistance/CoreDataStackTests.swift [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        XCTAssertEqual(twoSafesResultBeforeSave[1].name, "Safe 1")
+
+        // reset context; not saved object is discarded
+        context.reset()
+        let oneSafeResultAfterReseting = try context.fetch(Safe.allSafes())
+        XCTAssertEqual(oneSafeResultAfterReseting.count, 1)
+        XCTAssertEqual(oneSafeResultAfterReseting[0].name, "Safe 1")
+
+        // Delete entry
+        // NOTE: we can not use here 'newSafe1' variable as the context was reseted
```
Comment (DmitryBespalov) [inline MultisigTests/Persistance/CoreDataStackTests.swift:58]: grammar: reseted -> reset

File: MultisigTests/Persistance/CoreDataStackTests.swift [hunk 2]
```diff
@@ -0,0 +1,67 @@
+//
+//  SafeMOTests.swift
+//  MultisigTests
+//
+//  Created by Andrey Scherbovich on 15.04.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import XCTest
```
Comment (DmitryBespalov) [inline MultisigTests/Persistance/CoreDataStackTests.swift:9]: grammar: Persistance -> Persistence (folder name)

File: Multisig/Persistance/Multisig.xcdatamodeld/Multisig.xcdatamodel/contents [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="16119" systemVersion="19E266" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="Safe" representedClassName="Safe" syncable="YES" codeGenerationType="class">
+        <attribute name="address" optional="YES" attributeType="String"/>
+        <attribute name="confirmationsRequired" attributeType="Integer 32" defaultValueString="1" usesScalarValueType="YES"/>
+        <attribute name="contractVersion" optional="YES" attributeType="String"/>
+        <attribute name="ensName" optional="YES" attributeType="String"/>
+        <attribute name="fallbackHandler" optional="YES" attributeType="String"/>
+        <attribute name="masterCopyAddress" optional="YES" attributeType="String"/>
+        <attribute name="name" optional="YES" attributeType="String"/>
+        <attribute name="nonce" attributeType="Integer 32" defaultValueString="0" usesScalarValueType="YES"/>
```
Comment (DmitryBespalov) [inline Multisig/Persistance/Multisig.xcdatamodeld/Multisig.xcdatamodel/contents:11]: I'd make it Uint64

File: Multisig/Persistance/Multisig.xcdatamodeld/Multisig.xcdatamodel/contents [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="16119" systemVersion="19E266" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="Safe" representedClassName="Safe" syncable="YES" codeGenerationType="class">
+        <attribute name="address" optional="YES" attributeType="String"/>
+        <attribute name="confirmationsRequired" attributeType="Integer 32" defaultValueString="1" usesScalarValueType="YES"/>
```
Comment (DmitryBespalov) [inline Multisig/Persistance/Multisig.xcdatamodeld/Multisig.xcdatamodel/contents:5]: I would make it UInt64

File: Multisig.xcodeproj/xcshareddata/xcschemes/Multisig.xcscheme [hunk 5]
```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Scheme
+   LastUpgradeVersion = "1140"
+   version = "1.3">
+   <BuildAction
+      parallelizeBuildables = "YES"
```
Comment (DmitryBespalov) [inline Multisig.xcodeproj/xcshareddata/xcschemes/Multisig.xcscheme:6]: I don't understand why this file was added

File: MultisigTests/Persistance/CoreDataStackTests.swift (hunk 1 above)
Comment (sche) [reply MultisigTests/Persistance/CoreDataStackTests.swift:58]: 👍

File: MultisigTests/Persistance/CoreDataStackTests.swift (hunk 2 above)
Comment (sche) [reply MultisigTests/Persistance/CoreDataStackTests.swift:9]: 👍

File: Multisig/Persistance/Multisig.xcdatamodeld/Multisig.xcdatamodel/contents (hunk 3 above)
Comment (sche) [reply Multisig/Persistance/Multisig.xcdatamodeld/Multisig.xcdatamodel/contents:11]: Ok, I can do that

File: Multisig.xcodeproj/xcshareddata/xcschemes/Multisig.xcscheme (hunk 5 above)
Comment (sche) [reply Multisig.xcodeproj/xcshareddata/xcschemes/Multisig.xcscheme:6]: I added some CommandLineArgument to see sql queries in console, so this scheme became shared.

**Pull request merged**